scapy/layers/rtp.py

```diff
@@ -25,6 +25,8 @@
     31: 'H261',          32: 'MPV',
     33: 'MP2T',          34: 'H263' }
 
+extension_condition = lambda pkt: pkt.extension
```

Comment:
Please note that adding this will import `extension_condition` in the entire scapy (as you are located in a layer).

You might want to move this inside the RTP class ? Would be better i think